great_expectations/core/expectation_suite.py

```diff
@@ -642,7 +642,8 @@ def add_expectation_configurations(
         match_type: str = "domain",
         overwrite_existing: bool = True,
     ) -> List[ExpectationConfiguration]:
-        """
```

Comment:
@abegong Thank you for this -- I think that this same clarification should be added to `add_expectation()` and, possibly, to `_add_expectation()`.  Thanks!